rpc_jobs/multi_node_aio.yml

```diff
@@ -190,3 +208,154 @@
+              Deploy RPC w/ Script
+              Prepare MaaS
+              Setup MaaS
+              Verify MaaS
+              Install Tempest
+              Tempest Tests
+              Holland
+              Leapfrog Upgrade
+              Install Tempest
+              Tempest Tests
+              Holland
+              Verify MaaS
+              Prepare Kibana Selenium
+              Kibana Tests
+              Pause (use to hold instance for investigation before cleanup)
+              Cleanup
+              Destroy Slave
+
+    dsl: |
+      node(){{
+        try {{
+          dir("rpc-gating"){{
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            pubCloudSlave = load 'pipeline_steps/pubcloud.groovy'
+            common = load 'pipeline_steps/common.groovy'
+            multi_node_aio_prepare = load 'pipeline_steps/multi_node_aio_prepare.groovy'
+            deploy = load 'pipeline_steps/deploy.groovy'
+            maas = load 'pipeline_steps/maas.groovy'
+            tempest = load 'pipeline_steps/tempest.groovy'
+            horizon = load 'pipeline_steps/horizon.groovy'
+            kibana = load 'pipeline_steps/kibana.groovy'
+            holland = load 'pipeline_steps/holland.groovy'
+            github = load 'pipeline_steps/github.groovy'
+          }}
+          instance_name = common.gen_instance_name()
+          pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
+
+          node(instance_name){{
+            multi_node_aio_prepare.prepare()
+            deploy.deploy_sh(
+              vm: "deploy1",
+              environment_vars: [
+                "DEPLOY_HAPROXY=yes",
+                "DEPLOY_ELK=yes",
+                "DEPLOY_TEMPEST=no",
+                "DEPLOY_AIO=no",
+                "DEPLOY_MAAS=no"
+                ]
+            ) // deploy_sh
+            multi_node_aio_prepare.multi_node_aio_networking()
+            dir("rpc-gating"){{
+              // Checkout rpc-gating repo to avoid race conditions in parallel steps
+              git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            }}
+            parallel(
+              "maas": {{
+                maas.prepare(instance_name: instance_name)
+                maas.deploy("deploy1")
+                maas.verify("deploy1")
+              }},
+              "tempest": {{
+                tempest.tempest("deploy1")
+              }},
+              //"horizon": {{
+              //  horizon.horizon_integration()
+              //}}, kilo has its own horizon integration project from openstack
+             // "kibana": {{
+             //   kibana.kibana(env.KIBANA_SELENIUM_BRANCH, "deploy1")
+             // }}, dont think the kibana has kilo branch.
+              "holland": {{
+                holland.holland("deploy1")
+              }}
+            ) // parallel
+            deploy.upgrade_leapfrog(vm: "deploy1", environment_vars: environment_vars)
+            parallel(
```

Comment:
Don't bother with tests you do not already have. All you need to test for Kilo is the basic functionality. The work to implement some VM's, volumes, etc and monitor them through the leap upgrade process is enough of a test to validate that the installation is functional.